terraform/tfschema/validators_test.go

```diff
@@ -0,0 +1,86 @@
+/*
+   Copyright 2022 Gravitational, Inc.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package tfschema
+
+import (
+	"context"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/stretchr/testify/require"
+)
+
+func getObjectWithTwoKeys() types.Object {
+	return types.Object{
+		Attrs: map[string]attr.Value{
```

Comment:
It would be safer if AttrTypes would be defined here as well:
```
AttrTypes: map[string]attr.Type{
    "key1": types.String,
    "key2": types.String,
},
```